### PR TITLE
Skip failing test for web shard on Windows

### PR DESCRIPTION
**Pull request by jmagman**
Skip individual failing tests so the `web_tool_tests` shard can be turned on. https://github.com/flutter/flutter/issues/76360

Skip the expression evaluation test group.  Replace `batch1` and `batch2` functions with test groups.  Investigation in https://github.com/flutter/flutter/issues/76398

Skip hot reload tests.  Investigation in https://github.com/flutter/flutter/issues/76421

### CODE REVIEW

**Review by jonahwilliams — APPROVED** [review]
LGTM

**Comment by jmagman** [comment]
Just updated it to turn on in the try builder to validate in this PR that the rest of the shard passes on Windows.
Will mark the shard unflaky in prod in another PR once it goes green.

**Comment by jmagman** [comment]
All the run expression evaluation tests are timing out.

Swapped the `batch1` `batch2` functions to `groups`, and skip the entire `run` group.  Will see if the `test` ones also time out.

File: packages/flutter_tools/test/web.shard/expression_evaluation_web_test.dart [hunk 1]
```diff
@@ -14,215 +14,212 @@ import '../integration.shard/test_driver.dart';
...
-  final BasicProject _project = BasicProject();
-  Directory tempDir;
-  FlutterRunTestDriver _flutter;
-
-  Future<void> initProject() async {
-    tempDir = createResolvedTempDirectorySync('run_expression_eval_test.');
-    await _project.setUpIn(tempDir);
-    _flutter = FlutterRunTestDriver(tempDir);
+void main() {
+  Future<void> failToEvaluateExpression(FlutterTestDriver flutter) async {
```
Comment (jmagman) [inline packages/flutter_tools/test/web.shard/expression_evaluation_web_test.dart:18]: Moved these methods to the top of main.

This PR is easier to look at with whitespace hidden.

**Comment by jmagman** [comment]
Okay now `hot_reload_web_test` are timing out.  Skip those, too.

**Comment by jmagman** [comment]
Now web_run_test is timing out.  Maybe none of these tests pass, in which case the whole shard should be off.  This probably needs investigation by someone with a Windows machine.

**Comment by annagrin** [comment]
LGTM! I will look into the failing tests soon.

**Review by annagrin — APPROVED** [review]

**Comment by jmagman** [comment]
I'm closing this, I kept skipping them and the next one would time out. #76398 #76421

**Pull request closed without merging**